docs/esps/sendinblue.rst

```diff
@@ -130,12 +130,9 @@ SendinBlue can handle.
   Anymail has no way to communicate an attachment's desired content-type
   to the SendinBlue API if the name is not set correctly.
 
-**No attachments with templates**
-  If you are sending using a SendinBlue template, their API doesn't support ordinary
-  file attachments. Attempting to send an attachment with a template will result in the
-  SendinBlue API error message, "Please don't pass attachment content & templateId in same
-  request, instead use attachment url only."
-  See the :ref:`templates <sendinblue-templates>` section below.
+**Attachments using templates only with new template language**
+  If you are sending an email using a SendinBlue template,
+  you have to use the new template language in order to add file attachments.
```

Comment:
(We could probably just omit this: Anymail v7 or later requires the new template language no matter what, even without attachments.)

Reply:
Agreed and updated 👍